app/assets/stylesheets/provider/_typography.scss

```diff
@@ -37,9 +37,8 @@ $font-family-base:          RedHatDisplay, Overpass, overpass, helvetica, arial,
 // font-sizes
 $fontSize:                   1rem;
 
-$hero-title-font-size:       3 * $fontSize;
-$main-title-font-size:       2 * $fontSize;
-$secondary-title-font-size:  (3/2) * $fontSize;
+$main-title-font-size:       1.5rem; // --pf-c-content--h1--FontSize
```

Comment:
Would rem always be 16px?... I'm just wondering - I have no idea.

Reply:
`rem` is a relative to the root's element `font-size`. It's basically what was being achieved by using `n * $fontSize`... So as long as the initial font-size does not change, 1rem will be worth 8px.
NOTE: I haven't found where this 8px are defined, I know PF supports defining a custom `--pf-global--root--FontSize` that will apply to all rems.. but we don't use that.

Reply:
Hm... Last time I checked, rem is actually 16px, and it was the browser default (at least in Chrome)
So, main title would be 24px, which is what PF4 uses for h1 (so, it's OK).

Just wondering whether it makes sense using rem instead of px. But I guess it's fine.

Reply:
Yes, it's 16, sorry. I think we should use whatever PF uses.

Reply:
> Yes, it's 16, sorry. I think we should use whatever PF uses.

Yeah, that was my point. Why use `1.5rem` instead of `24px` (as PF4 defines it as `24px`) ?